addons/website/static/src/scss/website.scss

```diff
@@ -962,8 +968,8 @@ header {
                 flex: 0 0 auto !important;
                 overflow: auto;
                 flex-flow: column nowrap !important;
+                width: 100%;
```

Comment:
I don't understand this change... it forces the size of the off-canvas to 560px in normal desktop cases... :|

Reply:
> I don't understand this change... it forces the size of the off-canvas to 560px in normal desktop cases... :|

This was my mistake, I was too focused on the long links and I missed this. I have just made the correction.